### DISCUSSION TITLE
Better  update interactive

### DISCUSSION THREAD

**Discussion by JuanM04** [pnpm/pnpm#4394] [Ideas]
The `pnpm update -i` command could show both range and latest version in a glance, like Yarn does:

![`yarn upgrade-interactive`](https://user-images.githubusercontent.com/16712703/155819256-233fca0e-3017-4585-be81-250d6b256a35.png)

**Comment by zkochan** [comment]
I don't have objections if someone is willing to work on it.

**Comment by donfear** [comment]
hey, if anyone misses the old yarn upgrade-interactive, i made this tiny cli that does something similar for pnpm: https://github.com/donfear/pnpm-upgrade-interactive - lets you pick which dependencies to update in your workspaces with a simple interactive prompt. nothing fancy, but it's been helpful for me and a few friends so thought i'd share in case it helps anyone else too
